src/shared/components/Pagination/Pagination.tsx

```diff
@@ -27,33 +26,39 @@ const Pagination: React.FC<PaginationProps> = ({
 
   if (totalPages <= 1) return null
 
+  const handleChangePage = (pageItem: number | '...', idx: number) => {
+    if (pageItem !== '...') {
+      onChangePage(pageItem - 1)
+    } else {
+      // if pageItem === '...' switch to the next/prev page
+      onChangePage(idx === 1 ? internalPage - 2 : internalPage)
+    }
+  }
+
   return (
     <PaginationWrapper>
-      <PaginationButton
-        isHidden={internalPage <= 1}
-        isChevron={true}
+      <ChevronButton
```

Comment:
in designs icon has `gray[50]` color when `:active`. Or maybe it has that styles only on the right side at all times. Not sure about that.

Reply:
set default colors of the `<Button />`. Should be better now